source/print/zettlr-print-window.js

```diff
@@ -49,46 +50,78 @@ class ZettlrPrintWindow {
 
     // Also we need to know whether or not we should initiate in darkMode.
     let dm = url.searchParams.get('darkMode')
-    if (dm === 'true') $('body').addClass('dark')
+    if (dm === 'true') document.body.classList.add('dark')
 
     // Toggle the theme if there's an appropriate event
-    ipc.on('toggle-theme', (e) => { $('body').toggleClass('dark') })
+    ipc.on('toggle-theme', (e) => { document.body.classList.toggle('dark') })
 
     // activate event listeners for the window
     this._act()
   }
 
   init (name) {
     this._file = name
-    document.title = path.basename(name)
-    $('.title h1').text(path.basename(name))
-    $('.content').html(`<iframe src="${name}"></iframe>`)
+    let basename = path.basename(name)
+    document.title = basename
+
+    // Store some necessary values for later use
+    this._titleElem = document.querySelectorAll('.title')[0]
+
+    let title = document.querySelectorAll('.title h1')[0]
+    title.textContent = basename
+
+    this._iframe = document.createElement('iframe')
+    document.querySelectorAll('.content')[0].appendChild(this._iframe)
+    this._iframe.setAttribute('src', name)
     this._reposition() // Initial reposition
   }
 
   _act () {
     // Activate the window controls.
-    $('.windows-window-controls .close').click((e) => { ipc.send('message', { 'command': 'win-close', content: {} }) })
-    $('.windows-window-controls .resize').click((e) => { ipc.send('message', { 'command': 'win-maximise', content: {} }) })
-    $('.windows-window-controls .minimise').click((e) => { ipc.send('message', { 'command': 'win-minimise', content: {} }) })
+    let winClose = document.querySelectorAll('.windows-window-controls .close')[0]
```

Comment:
It seems like you are looking for only the first instance of the selector. Have you considered using [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector)?

```suggestion
    const winClose = document.querySelector('.windows-window-controls .close')
```

Also, you don't plan to reassign the type of the `winClose` variable, I recommend using `const` instead of `let`.